src/Migrations/Version20200619015717.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DoctrineMigrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20200619015717 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return '';
```

Comment:
(optional) may be could be added a little description

Reply:
You are right, working on ti!